content/src/service/synchronization/streaming/EventStreamProcessor.ts

```diff
@@ -102,7 +109,8 @@ export class EventStreamProcessor {
           }
           done()
         } catch (error) {
-          EventStreamProcessor.LOGGER.debug(
+          DCL_CONTENT_FAILED_DEPLOYMENTS_TOTAL.inc()
```

Comment:
Shouldn't this be together with the code that inserts the failed deployments in the database?

Reply:
The DB of failed entities seems empty, I want to measure how much times this path is being taken without explicitly inserting in the DB